src/app/@ansyn/menu-items/algorithms/components/tasks-form/tasks-form.component.ts

```diff
@@ -104,7 +104,8 @@ export class TasksFormComponent implements OnInit, OnDestroy {
 			} else {
 				return of(null);
 			}
-		})
+		}),
+		take(1) 
```

Comment:
why not on tasks service ? (take)

Reply:
I do not always invoke tasks service here - see the condition. In fact, in the situation that interested me here, I do not invoke the service (when the component is activated for a new task)